pywbem/cim_operations.py

```diff
@@ -6359,6 +6359,53 @@ def ModifyInstance(self, ModifiedInstance, IncludeQualifiers=None,
         (see :term:`DSP0200`). See :ref:`WBEM operations` for a list of all
         methods performing such operations.
 
+        The properties provided in the `ModifiedInstance` parameter specify
+        the new property values for the instance to be modified.
+
+        The `PropertyList` parameter determines the set of properties that are
+        designated to be modified. The WBEM server shall reject modification
+        requests for key properties.
```

Comment:
add 'or if properties are in the propertylist that are invalid for the target instance'

Reply:
"invalid for target instance" leaves some wiggle room,and this is a shall-rule. I prefer "not exposed by the creation class of the target instance."
DONE.